bin/openaps-install.sh

```diff
@@ -56,6 +56,12 @@ apt-get -o Acquire::ForceIPv4=true update && apt-get -o Acquire::ForceIPv4=true
 apt-get -o Acquire::ForceIPv4=true update && apt-get -o Acquire::ForceIPv4=true install -y sudo strace tcpdump screen acpid vim python-pip locate ntpdate ntp
 #check if edison user exists before trying to add it to groups
 
+# Install nvm and update to node 15.14.0 also update npm
+sudo curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+source ~/.bashrc
+nvm install 15.14.0
```

Comment:
Has this line been tested on a Pi 0 (ARMv6) or Edison (x86)?. I fully expect installing any major version 12 or higher (10 on x86) on such architectures with `nvm` will cause it to fall back to building from source. I would call that a non-starter, especially given how long the installation can be already, particularly on low power hardware like the Pi 0. I also haven't heard of or experienced great success with building from source.

I believe the necessity, then, to support older hardware, is to target older versions of node. Ideally we would just install the latest supported version of node, yes?

Out of curiosity, how did you come to the version 15.14.0 in your testing? Why not a lower or higher version?

Reply:
This was on a Pi Zero 2, so faster machine for sure.

Agree with it being better to install latest supported, again probably best managed through n.

I actually think I just picked this version up from a passing comment in the intend-to-bolus...not sure why I took it as gospel.